src/features/query-bar/QueryBar.js

```diff
@@ -27,12 +28,15 @@ const QueryBar = ({
     onFilterChange,
     onTemplateChange,
     templates,
-}: Props) => (
-    <section className="metadata-view-query-bar">
-        <TemplateButton activeTemplate={activeTemplate} onTemplateChange={onTemplateChange} templates={templates} />
-        <FilterButton columns={columns} conditions={conditions} onFilterChange={onFilterChange} />
-        <ColumnButton columns={columns} onColumnChange={onColumnChange} template={activeTemplate} />
-    </section>
-);
+}: Props) => {
+    const nonItemColumns = columns && columns.filter(column => !ITEM_PROPERTIES.includes(column.property));
```

Comment:
Should QueryBar do this or should the client pre-process before passing down? 

Having ITEM_PROPERTIES live here is odd to me. It means that QueryBar has awareness of something it need not be aware of.